catalog/app/containers/NavBar/Help.js

```diff
@@ -0,0 +1,313 @@
+import cx from 'classnames'
+import * as Lab from '@material-ui/lab'
```

Comment:
`lab` should come after `core` (A-Z) and scoped packages should come after non-scoped